src/components/SidePanels/TxHistoryRow.vue

```diff
@@ -45,17 +45,29 @@ import getMemoFromByteString from '@/services/history/utils'
 })
 export default class TxHistoryRow extends Vue {
     @Prop() transaction!: ITransactionData
+    helpers = this.globalHelper()
+    navigate(to: string) {
+        this.helpers.navigate(to)
+    }
+    openInNewTab(url: string) {
+        window.open(url, '_blank', 'noreferrer')
+    }
 
-    get explorerUrl(): string | null {
+    get explorerUrl(): string {
         let network: AvaNetwork = this.$store.state.Network.selectedNetwork
         let chains = this.$store.state.History.chains
-        if (network.explorerSiteUrl && chains.length > 0) {
+        if (chains.length > 0) {
             let alias = chains?.find((elem: Chain) => elem.chainID === this.transaction.chainID)
                 .chainAlias
-            let url = `${network.explorerSiteUrl}/${alias}-chain/transactions/${this.transaction.id}`
-            return url
+            if (network.explorerSiteUrl) {
```

Comment:
https://eips.ethereum.org/EIPS/eip-3091 this should be a standard, we mentioned it earlier, if it's introduced on our site, it would ease up some stuff. Idk how when it comes to c/p/x chain

Reply:
I agree, will be updating our explorer URLs to use these standards.

Reply:
@Noctunus do we have to support the routing for Avalanche explorer because it's kind of a mess to handle

C-Chain: `https://snowtrace.io/tx/:hash`
P-Chain: `https://subnets.avax.network/p-chain/tx/:hash`
X-Chain: `https://explorer-xp.avax.network/tx/:hash`

Should I create a function that handles these routing for Avalanche, or not?